www/artists/dloran/work.html

```diff
@@ -25,11 +25,19 @@
                 </p>
             </div>
             <div class="ContentFlow">
-                <div class="loadIndicator">
-                    <div class="indicator"></div>
-                </div>
+                <div class="loadIndicator"><div class="indicator"></div></div>
                 <div class="flow">
-                    <img class="item" src="images/david.jpg" title="David Loran in front of his mural behind Gales Flourists" alt="no alt text"> <img class="item" src="images/fish.jpg" title="This is a mural is on the back of Brandees and it is a lionfish" alt="no alt text"> <img class="item" src="images/flower.jpg" title="This is a mural he was asked to do on the back of Gales flourists downtown" alt="no alt text"> <img class="item" src="images/girl.jpg" title="This mural is behind macs " alt="no alt text"> <img class="item" src="images/mural.jpg" title="This is a mural he did on the inside of the restaurant the deli llama that is in the scarth street mall" alt="no alt text"> <img class="item" src="images/mural3.jpg" title="This is a mural he did on the back of his old shed" alt="no alt text"> <img class="item" src="images/mural4.jpg" title="This mural is on the back of vintage vinyl which is in the downtown area" alt="no alt text"> <img class="item" src="images/mural5.jpg" title="This is a mural he did on the side of a tattoo shop in Regina" alt="no alt text"> <img class="item" src="images/mural6.jpg" title="This is a mural he did for the old souls sneaker boutique that is in the cathedral area in Regina" alt="no alt text"> <img class="item" src="images/mural8.jpg" title="This mural is at the mercury,a restaurant downtown" alt="no alt text"> <img class="item" src="images/mural9.jpg" title="This is a mural the 11 hooks crew did at the yearly arts festival in Regina" alt="no alt text">
```

Comment:
Thank you for adding new lines between these images. It may not be strictly necessary, but it is definitely appreciated.